src/site/stages/build/drupal/graphql/healthServicesListingPage.graphql.js

```diff
@@ -120,7 +120,7 @@ function getNodeHealthServicesListingPages(operationName, offset, limit) {
       nodeQuery(
         limit: ${limit}
         offset: ${offset}
-        sort: { field: "title", direction:  ASC }
+        sort: { field: "nid", direction:  ASC }
```

Comment:
Does this alter the display order, or are these not related?

Reply:
There is logic after the GraphQL response that should do the actual sort. This query didn't have a sort order at all before so I hope we weren't trying to rely on it.